test/validation/api/atomic/atomic.c

```diff
@@ -62,6 +63,10 @@ typedef struct {
 static odp_shm_t global_shm;
 static global_shared_mem_t *global_mem;
 
+/* Declare a global odp_u128_t variable for use in odp_atomic_init_u128() and
+ * odp_atomic_store_u128(). */
+static odp_u128_t a128u_tmp;
```

Comment:
Looks to me this doesn't have to be global and anyway all global data should go inside `global_shared_mem_t`.

Reply:
Okay, I will remove it. Thanks!